src/index.ts

```diff
@@ -48,6 +48,39 @@ function setupOpenMRS() {
             moduleName: "@openmrs/esm-patient-chart-widgets"
           }
         )
+      },
+      {
+        id: "attachments-overview-widget",
+        slot: "attachments-overview-tab",
+        load: getAsyncLifecycle(
+          () => import("./widgets/attachments/attachments-overview.component"),
+          {
+            featureName: "attachments",
+            moduleName: "@openmrs/esm-patient-chart-widgets"
```

Comment:
The `moduleName` can be placed in a local variable `const moduleName = ...` and could be reused.